example/windows/SliderTestWindows/App.cpp

```diff
@@ -0,0 +1,42 @@
+#include "pch.h"
+
+#include "App.h"
+
+#include "AutolinkedNativeModules.g.h"
+#include "ReactPackageProvider.h"
+
+
+using namespace winrt::SliderTestWindows;
+using namespace winrt::SliderTestWindows::implementation;
+
+/// <summary>
+/// Initializes the singleton application object.  This is the first line of
+/// authored code executed, and as such is the logical equivalent of main() or
+/// WinMain().
+/// </summary>
+App::App() noexcept
+{
+    MainComponentName(L"example");
+
+#if BUNDLE
+    JavaScriptBundleFile(L"index.windows");
+    InstanceSettings().UseWebDebugger(false);
```

Comment:
We removed this pattern at some point - not super recently but in the last few months - and we now use a MainPage component instead which is how the usual UWP app template works in VS, is this not using RNW 62+?

Reply:
I was using RNW62 and this is what got generated

Reply:
The same thing got generated for Tatiana: https://github.com/react-native-community/progress-view/pull/37/files#diff-c0553c5c1db81914e0098a7cf30ac5c3

Reply:
Can you point me to the updated template you're referencing?

Reply:
@rectified95 https://github.com/microsoft/react-native-windows/blob/master/vnext/local-cli/templates/cpp/src/App.cpp
https://github.com/microsoft/react-native-windows/blob/master/vnext/local-cli/templates/cpp/src/MainPage.cpp

Reply:
Thanks for the pointer,
I've looked at those files and they contain elements (eg. xamlNamespaceCpp) that get replaced by the CLI during project creation. It'd be difficult for me to replace them all manually.

This is just an example app, so I think it should be ok? Especially given that this is what was generated for me on RNW 62.